data/UbuConKorea2023.yml

```
@@ -0,0 +1,7 @@
+name: UbuCon Korea 2023
+description: 한국 우분투 사용자 컨퍼런스
+thumbnail: https://2023.ubuntu-kr.org/logo.svg
+date: 2023-09-09
+location: 서울 종로구 종로1길 50 더케이트윈타워 A동 11-16층
+link: 
+  UbuCon Korea 2023: https://2023.ubuntu-kr.org/ko/
```

버튼명이 `UbuCon Korea 2023`면 조금 어색할 거 같은데 혹시 `홈페이지` 같은 이름으로 바꾸는 건 어떤가요?
```suggestion
  홈페이지: https://2023.ubuntu-kr.org/ko/
```

---

앗ㅜ죄송 이제 봤네요 ;;
저도 동의합니다!
좋은 제안 감사드려요!